core/primitives/src/shard_layout.rs

```diff
@@ -547,4 +563,20 @@ mod tests {
     fn parse_account_ids(ids: &[&str]) -> Vec<AccountId> {
         ids.into_iter().map(|a| a.parse().unwrap()).collect()
     }
+
+    fn print_shard_layout(name: &str, shard_layout: &ShardLayout) {
+        println!("{name}");
+        println!("{}", serde_json::to_string(shard_layout).unwrap());
+    }
+
+    #[test]
```

Comment:
Is this something you want to keep? Maybe include the json strings and assert equality? (Like use the insta assert snapshot thing)